mee/times.h

```diff
@@ -0,0 +1,16 @@
+/* Copyright 2018 SiFive, Inc */
+/* SPDX-License-Identifier: Apache-2.0 */
+
+#ifndef MEE__TIMES_H
+#define MEE__TIMES_H
+
+/* Read the machine cycle counts from csrr register */
+int mee_get_cyclecount(unsigned long *mcc);
```

Comment:
Since these functions are public they need to be documented as part of the API docs in `doc/`.